app/src/play/java/chat/rocket/android/analytics/GoogleAnalyticsForFirebase.kt

```diff
@@ -35,19 +35,31 @@ class GoogleAnalyticsForFirebase @Inject constructor(val context: Context) :
     }
 
     // WIDECHAT tracking BSSID
-    override fun logMessageSent(event: SubscriptionTypeEvent, serverUrl: String) {
+    override fun logMessageSent(messageType: String, serverUrl: String) {
         var bssid = SharedPreferenceHelper.getString(Constants.CURRENT_BSSID, "none")
         if (bssid == "none") {
             bssid = SharedPreferenceHelper.getString(Constants.LOCATION_PERMISSION, "none")
 
         }
         firebaseAnalytics.logEvent("message_sent", Bundle(2).apply {
-            putString("subscription_type", event.subscriptionTypeName)
-//            putString("server", serverUrl)
+            putString("message_type", messageType)
             putString("wifi_bssid", bssid)
         })
     }
 
+    override fun logSendMessageException(countToSend: Int, exceptionDescription: String, serverUrl: String) {
+        var bssid = SharedPreferenceHelper.getString(Constants.CURRENT_BSSID, "none")
+        if (bssid == "none") {
+            bssid = SharedPreferenceHelper.getString(Constants.LOCATION_PERMISSION, "none")
+        }
+        val params = Bundle()
```

Comment:
should we revert this different way of doing Bundle since the issue turned out to be the length and not that there was something wrong with how the params were being put in?

Reply:
yes